shared/templates/file_permissions/ansible.template

```diff
@@ -6,7 +6,16 @@
 
 {{% for path in FILEPATH %}}
 {{% if IS_DIRECTORY %}}
-{{% if FILE_REGEX %}}
+
+{{%- if FILE_REGEX %}}
+{{% set STATE="file" %}}
+{{% set FIND_TYPE="-type f" %}}
+{{% set FIND_FILE_REGEX="-regex FILE_REGEX[loop.index0]" %}}
```

Comment:
The CI error seems legit and I believe is caused by this line. The final Ansible task using this `FILE_REGEX[loop.index0]` literally:
```
- name: Find /etc/audit/ file(s)
   command: find -H /etc/audit/ -maxdepth 1 -perm /u+xs,g+xws,o+xwrt  -type f -regex FILE_REGEX[loop.index0]
   register: files_found
   changed_when: false
   failed_when: false
   check_mode: false
...
```
You can try:
`{{% set FIND_FILE_REGEX="-regex \"" ~ FILE_REGEX[loop.index0] ~ "\"" %}}`

Reply:
@marcusburghardt thanks! fixed it, but still you might want to check my comment as I think that rule is not working as expected.